python/pyarrow/jvm.py

```diff
@@ -36,12 +36,12 @@ def jvm_buffer(arrowbuf):
     ----------
 
     arrowbuf: io.netty.buffer.ArrowBuf
-        Arrow Buffer representation on the JVM
+        Arrow Buffer representation on the JVM.
 
     Returns
     -------
-    pyarrow.Buffer
-        Python Buffer that references the JVM memory
+    pyarrow.Buffer :
```

Comment:
```suggestion
    pyarrow.Buffer
```

(if there is no name, I thought the colon was not needed)

Reply:
Fixed.